packages/components/src/custom-gradient-picker/style.scss

```diff
@@ -102,3 +97,18 @@ $components-custom-gradient-picker__padding: 6px; // 36px container, 24px handle
 		}
 	}
 }
+
+// All these styles should be made generic and changed on the inputs for all components.
```

Comment:
I appreciate the "todo". It's something we might want to look into soon.

Reply:
Agreed. We should look into all control/input components and standardize their dimensions and spacing throughout the library of `@wordpress/components`